packages/react-composites/src/composites/CallComposite/pages/NoticePage.tsx

```diff
@@ -31,6 +31,7 @@ export interface NoticePageProps {
  */
 export function NoticePage(props: NoticePageProps): JSX.Element {
   const adapter = useAdapter();
+  const isPSTNorACSCall = adapter.getState().isPSTNorACSCall;
```

Comment:
Wouldn't recommend adding this attribute to adapter state since it will become publicly available
Could we track it internally in composites somehow?

Reply:
wouldn't we want to show this publicly in the adapter? this state could be useful to contoso to have their app do something special based on the type of call. we do also do the same thing with the `isTeamsCall` prop, its just being set in the constructor though.

Reply:
Ideally, such flags should be coming directly from the headless SDK as they are the ultimate source of truth

Reply:
I'd also like to avoid it being a public property - ideally contoso either would know as they pass in the locator, or they shouldn't care...
But our design means that the composite should only be using public adapter props so not sure we can avoid that...

Reply:
(my vote would be to go with the other PR that removes the rejoin button and we do a major version bump)